airbyte-protocol/protocol-models/src/main/java/io/airbyte/protocol/models/CatalogHelpers.java

```diff
@@ -334,10 +335,15 @@ private static UpdateStreamTransform getStreamDiff(final AirbyteStream streamOld
     final Set<FieldTransform> fieldTransforms = new HashSet<>();
     final Map<List<String>, JsonNode> fieldNameToTypeOld = getFullyQualifiedFieldNamesWithTypes(streamOld.getJsonSchema())
         .stream()
-        .collect(Collectors.toMap(Pair::getLeft, Pair::getRight));
+        .collect(
+            () -> new HashMap<>(),
```

Comment:
Nit:  I think that you can do `HashMap::new` here as well.

Reply:
Done